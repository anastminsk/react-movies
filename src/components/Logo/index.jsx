import React from 'react';
import styled from 'styled-components';

const LogoComponent = ({ className }) => (
	<span className={className}>
		<b>netflix</b>roulette
	</span>
);

const Logo = styled(LogoComponent)`
	font-size: 20px;
	color: #f65261;
	letter-spacing: 1px;
`;

export default Logo;
