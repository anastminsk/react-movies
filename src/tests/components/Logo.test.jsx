import { render } from '@testing-library/react';
import React from 'react';

import Logo from '../../components/Logo';

describe('Logo', () => {
	test('renders Logo snapshot', () => {
		const { asFragment } = render(<Logo />);
		expect(asFragment()).toMatchSnapshot();
	});
});
