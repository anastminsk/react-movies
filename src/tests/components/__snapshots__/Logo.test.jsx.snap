// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Logo renders Logo snapshot 1`] = `
<DocumentFragment>
  <span
    class="Logo-sc-95wlsg-0 jJNJyJ"
  >
    <b>
      netflix
    </b>
    roulette
  </span>
</DocumentFragment>
`;
